test/functional/page_objects/common_page.js

```diff
@@ -186,12 +186,12 @@ export function CommonPageProvider({ getService, getPageObjects }) {
                   }
                 });
             });
-          })
-          .then(async () => {
-            if (appName === 'status_page') return;
-            if (await testSubjects.exists('statusPageContainer')) {
-              throw new Error('Navigation ended up at the status page.');
-            }
+          // })
+          // .then(async () => {
+          //   if (appName === 'status_page') return;
+          //   if (await testSubjects.exists('statusPageContainer', 500)) {
+          //     throw new Error('Navigation ended up at the status page.');
+          //   }
           });
```

Comment:
This ^ is a check to see if Kibana had a problem and took you to the status page.  I commented it out because it was always taking 10 seconds.